tests/foreman/cli/test_host.py

```diff
@@ -3321,3 +3321,23 @@ def test_positive_errata_list_of_sat_server(self):
         hostname = ssh.command('hostname').stdout[0]
         host = Host.info({'name': hostname})
         self.assertIsInstance(Host.errata_list({'host-id': host['id']}), list)
+
+
+class EncDumpTestCase(CLITestCase):
+    """Tests for Dump host's ENC YAML"""
+
+    @tier1
+    def test_positive_dump_enc_yaml(self):
+        """Dump host's ENC YAML. Check BZ for details.
+
+        :id: 50bf2530-788c-4710-a382-d034d73d5d4d
+
+        :expectedresults: Dump host's ENC YAML
+
+        :BZ: 1372731
+
+        :CaseImportance: Critical
+        """
+        hostname = ssh.command('hostname').stdout[0]
+        host = Host.info({'name': hostname})
+        self.assertIsInstance(Host.enc_dump({'name': host['name']}), list)
```

Comment:
can we assert on some values from that output ?

Reply:
Any way to cover this bug we need only that the the enc-dump command does not fail perhaps change the test doc string and :expectedresults:
like:
```
:expectedresults: Ensure that enc-dump does not fail
```